packages/block-library/src/navigation-menu/editor.scss

```diff
@@ -1,3 +1,60 @@
+// Reduce the paddings, margins, and UI of inner-blocks.
+// @todo: eventually we may add a feature that lets a parent container absorb the block UI of a child block.
+// When that happens, leverage that instead of the following overrides.
+[data-type="core/navigation-menu"] {
```

Comment:
We should coordinate the work into adding props/flags to inner blocks that provide customization options and avoid the need for these styles.

Reply:
Yep, this is ticketed in https://github.com/WordPress/gutenberg/issues/17267. But it's also necessary for us to be able to move forward, without waiting for that ticket to be worked on. What do you think?

Reply:
Yes, I think for now we can keep these styles.